python/cudaq/__init__.py

```diff
@@ -39,3 +42,68 @@
     initKwargs['target'] = sys.argv[sys.argv.index('--target') + 1]
 
 initialize_cudaq(**initKwargs)
+
+# Expose global static quantum operations
+h = h()
+x = x()
+y = y()
+z = z()
+s = s()
+t = t()
+
+rx = rx()
+ry = ry()
+rz = rz()
+r1 = r1()
+swap = swap()
+
+
+class MidCircuitMeasurementAnalyzer(ast.NodeVisitor):
+    """The `MidCircuitMeasurementAnalyzer` is a utility class searches for 
+       common measurement - conditional patterns to indicate to the runtime 
+       that we have a circuit with mid-circuit measurement and subsequent conditional 
+       quantum operation application."""
+
+    def __init__(self):
+        self.measureResultsVars = []
+        self.hasMidCircuitMeasures = False
+
+    def visit_Assign(self, node):
+        target = node.targets[0]
+        if not 'func' in node.value.__dict__:
+            return
+        creatorFunc = node.value.func
+        if 'id' in creatorFunc.__dict__ and creatorFunc.id == 'mz':
```

Comment:
Does this need to include `mx` and `my`, too?

Reply:
yes